doc/user/content/concepts/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Concepts"
+description: "Core Concepts in Materialize"
+disable_list: true
+menu:
+  main:
+    parent: get-started
+    weight: 25
+    identifier: concepts
+aliases:
+  - /overview/api-components/
+  - /overview/key-concepts/
+  - /get-started/key-concepts/
+---
+
+The pages in this section introduces some of the key concepts in Materialize:
+
+Component                                | Use
+-----------------------------------------|-----
```

Comment:
Updated the links to new pages.
Consolidated Views and Mat Views into a single row.